app/assets/stylesheets/default.scss

```diff
@@ -0,0 +1,20 @@
+.pageBackground{
+    background: #F9C784;
+}
+.pizzaContainer{
+    background: #FAA300;
+    margin-top: 1.5rem;
+}
+
+.pizzaCard{
+    background: #FF6201;
+    $card-background: #FF6201;
+    border: 10px, solid, #FF6201;
+    border-radius: 50%;
+}
+
+.pizzaImage{
+    height: 75%;
+    width: auto;
+    max-width: 500px;
+}
```

Comment:
It would be awesome to see a space between your classes and the bracket. Makes it easer to read.